ambry-server/src/test/java/com/github/ambry/server/AmbryServerRequestsTest.java

```diff
@@ -318,7 +318,7 @@ public void scheduleCompactionFailureTest() throws InterruptedException, IOExcep
    * @throws InterruptedException
    * @throws IOException
    */
-  @Test
+  //@Test
```

Comment:
Why comment it?

Reply:
Good catch, it was previously disabled for debugging. I will remove it. Thanks.